rosplan_dispatch_msgs/SensorDispatch action
---
bool sensor_success
